pkgs/development/libraries/x264/default.nix

```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "lib" "dev" ];
 
-  preConfigure = lib.optionalString (stdenv.buildPlatform.isx86_64 || stdenv.hostPlatform.isi686) ''
+  preConfigure = lib.optionalString (stdenv.buildPlatform.isx86_64 || stdenv.hostPlatform.isi686 || stdenv.hostPlatform.isAarch32) ''
     # `AS' is set to the binutils assembler, but we need nasm
     unset AS
   '' + lib.optionalString (stdenv.hostPlatform.isAarch64 || stdenv.hostPlatform.isAarch32) ''
```

Comment:
Maybe remove `stdenv.hostPlatform.isAarch32` from this code branch if it is already handled above?

Reply:
Looks like it's not even needed, I've picked this from older tree and didn't notice it was fixed few lines bellow as I didn't get any conflicts :) Good catch!